packages/style/package.json

```diff
@@ -57,7 +57,9 @@
         "merge-stream": "1.0.1",
         "mini-css-extract-plugin": "0.12.0",
         "minimist": "1.2.5",
-        "postcss-loader": "3.0.0",
+        "node-sass": "5.0.0",
```

Comment:
`node-sass` was replaced with `sass` (see line 64)

Reply:
the pr is not merge yet so I left it there :see_no_evil: